docs/source/science_examples.rst

```diff
@@ -6,12 +6,12 @@ Science Examples
   :caption: Science Examples:
 
 
-  science/HLS/HLS_search.ipynb
-  science/GEDI/GEDI02_A_tutorial.ipynb
-  science/GEDI/SearchGEDI.ipynb
+  science/HLS/HLS30.ipynb
```

Comment:
HLSL30.ipynb

(I added the L at the end of HLS because it used Landsat data. Figured that might would help some with naming if we decided to add more HLS tutorials.)